webapp/src/main/resources/properties/en.properties

```diff
@@ -718,6 +718,12 @@ workbench.translationHistoryModal.title=Translation History
 # Label for the button to close the translation history modal dialog
 textUnit.translationHistoryModal.close=Close
 
+# Text telling the user why one of the rows is highlighted
+textUnit.translationHistoryModal.currentUnitExplanation=The highlighted row is the currently approved translation.
```

Comment:
Instead of `currently approved translation`, use `current translation`. We sometimes have translation with `Needs Review` or `Rejected` status.